### ISSUE TITLE
go command required, not found: exec: "go"

### ISSUE THREAD

**Issue by MasterMeng** [securego/gosec#548]
### Steps to reproduce the behavior
Call gosec in source code, build to exec, delete go from $PATH, get error:"err: go command required, not found: exec: "go": executable file not found in $PATH: stderr: "
### gosec version
v2.5.0
### Go version (output of 'go version')
go 1.15
### Operating system / Environment
CentOS 8

**Comment by ccojocar** [comment]
Is the `go` installed in your PATH? It seems that cannot be found. This issue is not related to gosec. I am going to close it.

**Comment by MasterMeng** [comment]
In my code, I have used some functions in gosec,such as gosec.NewImportTracker(),gosec.NewAnalyzer(). Then built it and move the binery to a new operating system without GOPATH,  I got such error "err: go command required, not found: exec: "go": executable file not found in $PATH: stderr: " after run.  Without gosec code in my code, it can work without GOPATH.  

I want to known if must have GOPATH when I use gosec in my program.
